Cargo.toml

```diff
@@ -28,6 +28,7 @@ serde_json = "1"
 tokio = "1.42"
 tokio-stream = "0.1.16"
 tonic = { version = "0.12.3", default-features = false }
+tonic-012 = { package = "tonic", version = "0.12.3", default-features = false }
```

Comment:
out of curiosity, why the duplication here ? Is it solely to have the error crate's dependency separated from the other crates' ?

Reply:
I want to have versionned dependency in `twirp-error` because `Tonic` is a public dependency and we might want to support multiple Tonic versions at the same time and not do a breaking release each time Tonic does.

However, Twurst generate code with tonic-build to test the gRPC server. So I need to have a dependency named "tonic". And because it is used in test or in the internal server implementation I don't care about versioning.